### PR TITLE
feature(credit transaction db):credit transaction db

### PR DESCRIPTION
**Pull request by dharmykoya**
#### What does this PR do?
to convert the credit transaction endpoint from the data structure to database
change test functions to use database 
#### Description of Task to be completed?
- created  table transactions
- rewrite the credit transaction methods
- rewrite the transaction type method
- add a new SQL to insert into transactions table 
- change test case to use DB data
- add a new SQL to update the accounts table
## Installation
1. Ensure you have Node.js and npm installed

2. Clone this repo
```bash
$ git clone https://github.com/dharmykoya/book_a_meal.git
```
3. Install Dependencies
```bash
npm install
```
4. Run Test
```bash
npm run test-db
```
#### Any background context you want to provide?
Javascript, Nodejs
#### What are the relevant pivotal tracker stories?
[#165419184](https://www.pivotaltracker.com/story/show/165419184)
#### Screenshots (if appropriate)
NA
#### Questions:
NA

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ cache:
     - "server/node_modules"
 before_script:
     - psql -c 'create database bankaa;' -U postgres
-    - npm run test-db    
+   # - npm run test-db
```
Comment (samdiano) [inline .travis.yml]: what's the # for?

Comment (dharmykoya) [reply .travis.yml]: to comment out the line. I found out while it builds on Travis it runs the command twice

Comment (dharmykoya) [reply .travis.yml]: should I remove the line and commit again?

Comment (samdiano) [reply .travis.yml]: yeah... remove it

**Review by samdiano — APPROVED** [review]

**Pull request merged**